src/test/java/uk/nhs/prm/deduction/e2e/transfer_tracker_db/TransferTrackerDbMessage.java

```diff
@@ -0,0 +1,49 @@
+package uk.nhs.prm.deduction.e2e.transfer_tracker_db;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.gson.GsonBuilder;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TransferTrackerDbMessage {
+
+    private String conversationId;
+    private String largeEhrCoreMessageId;
+    private String nemsMessageId;
+    private String nhsNumber;
+    private String sourceGp;
+    private String state;
+    private String nemsEventLastUpdated;
+    private String createdAt;
+    private String lastUpdatedAt;
+
+    public TransferTrackerDbMessage(@JsonProperty("conversationId") String conversationId,
```

Comment:
Why are we specifying @JsonProperty when the field name is the same as the JsonProperty name? Unless there's a reason here that I'm missing, Jackson should be able to do this automatically and it'll look cleaner.